specs/pipe_analysis/coaddAnalysis/rhoStatistics.yaml

```diff
@@ -0,0 +1,116 @@
+# Rho statistics from pipe_analysis
+#
+# Note that the PSF ellipticity residuals represented by the Rho 1 statistic
+# are not included here, but they are captured by the TE1 and TE2 metrics
+# (located in `verify_metrics/specs/validate_drp\TEx.yaml`).
+
+---
+# Specification partial
+id: "RhoStat_smallScale-base"
```

Comment:
I was looking for consistency as well.  If all can be referred to (here and in the metric names and definitions) as `rhoStat`, then use that everywhere.  If you feel that the abbreviation is not appropriate, the they should all be `rhoStatistics`.  A mix-n-match is just confusing.

Reply:
Agreed. I use `rhoStatistics` in full everywhere in this file.